### ISSUE TITLE
Compressor isn't minifying assets as intended on live site

### ISSUE THREAD

**Issue by emilyhorsman** [littleweaver/littleweaverweb.com#39]
(Obviously trimmed) CSS:

```
/* line 9, /var/www/env/local/lib/python2.7/site-packages/bootstrap/static/bootstrap/_normalize.scss */
html {
  font-family: sans-serif;
  -ms-text-size-adjust: 100%;
  -webkit-text-size-adjust: 100%; }

/* line 19, /var/www/env/local/lib/python2.7/site-packages/bootstrap/static/bootstrap/_normalize.scss */
body {
  margin: 0; }
```

>Minifying the following CSS resources could reduce their size by 165.9KiB (40% reduction).

**Comment by melinath** [comment]
The comments are no longer included in the css, maybe because we turned off debug? There's still whitespace, but I'm not sure I believe that minification has significant gains over gzip when it comes to whitespace removal?

**Comment by melinath** [comment]
... but I also think that amazon s3 may not be gzipping content.

**Comment by chigby** [comment]
Documentation on this subject is slightly difficult to find, but it looks like our culprit might be here: https://bitbucket.org/david/django-storages/src/f153a70ba254dc129d9403546809a02256ef75b5/storages/backends/s3boto.py?at=default&fileviewer=file-view-default#s3boto.py-235

That is, we have to set `AWS_IS_GZIPPED = True` in our settings. 

Is there a good way to test this out without potentially breaking all of our css? @melinath Do we still have a staging server for this?

**Comment by melinath** [comment]
Resolved with merge of #78
